scannerlib/configchecks/group_criteria.go

```diff
@@ -215,3 +223,65 @@ func getCmpStr(gc *ipb.GroupCriterion) string {
 	}
 	return strconv.Itoa(int(gc.GetConst()))
 }
+type lessThanVersionMatcher struct {
+	cmpStr string
+}
+func (m *lessThanVersionMatcher) match(group string) bool {
+	if !swVersRe.MatchString(group) {
+		log.Printf("unable to parse %q as a software version", group)
+		return false
+	}
+	version := strings.ReplaceAll(m.cmpStr, "-", ".")
```

Comment:
Why do we need to do this replacement? Can you add a comment about it? 
AFAICS we'll split by \\W+ so this shouldn't be needed to compare the digits?